src/org/centos/pipeline/PackagePipelineUtils.groovy

```diff
@@ -475,3 +475,148 @@ def setScratchVars(Map parsedMsg) {
         env.request_1 = parsedMsg['request'][1]
     }
 }
+
+/**
+ * Based on tagMap, add comment to GH with
+ * instructions to manual commands
+ *
+ * @param map of tags
+ * @return
+ */
+def sendPRCommentforTags(imageOperationsList) {
```

Comment:
You may be able to use
https://github.com/openshift/contra-lib/blob/master/src/org/centos/contra/pipeline/Utils.groovy#L688
instead and not maintain this function

Reply:
The function from contra-lib sends IRC notification, this one if I got it correctly updates the PR with a comment about the image that got updated, right?

Reply:
Yep, you're right, my mistake